docs/ui/Do.tsx

```diff
@@ -0,0 +1,30 @@
+import { Text } from '@marigold/components';
+
+import { Image } from '@/ui/Image';
+
+interface Props {
+  img: string;
+  description: string;
+}
+
+export const Do = ({ description, img }: Props) => {
+  return (
+    <div className="border-border flex flex-col border">
+      <Image width={700} height={900} src={img} alt="best practice" />
```

Comment:
Maybe this can be a `children` so we can use something different here if we want to